.github/workflows/r-ci.yml

```diff
@@ -91,6 +87,13 @@ jobs:
       - name: Update Packages
         run: Rscript -e 'update.packages(ask=FALSE)'
 
+      - name: Re-install 0.22 build of tiledb-r with core 2.18
+        run: |
+          cd apis/r
+          wget https://github.com/TileDB-Inc/TileDB-R/archive/refs/tags/0.22.0.tar.gz
+          Rscript -e 'remotes::install_deps("0.22.0.tar.gz")'
+          R CMD INSTALL 0.22.0.tar.gz    # as 0.22.0 is needed by SOMA 1.6
```

Comment:
You should not need to do it twice, you can can skip the `update.packages(ask=FALSE)` too.  That was from a different time fixing a different issue.

Reply:
@eddelbuettel I went through multiple CI iterations on #1972 on the `release-1.5` branch to find, and use, a known-good pattern which we could then apply for subsequent PRs on subsequent branches.

This PR is a such a subsequent PR on a subsequent branch (`release-1.6`).